packages/xstate-lit/test/UseActor.ts

```diff
@@ -0,0 +1,80 @@
+import { html, LitElement } from 'lit';
+import type { AnyMachineSnapshot } from 'xstate';
+import { fromPromise } from 'xstate/actors';
+import { fetchMachine } from './fetchMachine.ts';
+import { UseMachine } from '../src/index.ts';
+
+const onFetch = () =>
+  new Promise<string>((res) => {
+    setTimeout(() => res('some data'), 50);
+  });
+
+const fMachine = fetchMachine.provide({
+  actors: {
+    fetchData: fromPromise(onFetch)
+  }
+});
+
+export class UseActor extends LitElement {
+  fetchController: UseMachine<typeof fetchMachine> = {} as UseMachine<
+    typeof fetchMachine
+  >;
```

Comment:
wouldn't it be possible to call `new UseMachine(...)` here? declaring a property like this would be much easier for consumers

Reply:
Yes, updated.